Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -1332,17 +1328,11 @@ def modify_user_ou_command(default_base_dn):
 
     user_name = args.get('user-name')
     dn = user_dn(user_name, args.get('base-dn') or default_base_dn)
-
-    success = conn.modify_dn(dn, "CN={}".format(user_name), new_superior=args.get('full-superior-dn'))
+    success = modify_user_ou(dn, new_ou=args.get('full-superior-dn'))
     if not success:
         raise Exception("Failed to modify user OU")
 
-    demisto_entry = {
-        'ContentsFormat': formats['text'],
-        'Type': entryTypes['note'],
-        'Contents': "Moved user {} to {}".format(user_name, args.get('full-superior-dn'))
-    }
-    demisto.results(demisto_entry)
+    return f'Moved user {user_name} to {args.get("full-superior-dn")}'
```

Comment:
Why did we change here the demisto.results to a string. 
Better to return CommandResults if possible.

Reply:
No need to create an `CommandResults` instance for that